src/pages/Embeds/Plio.vue

```diff
@@ -656,12 +700,53 @@ export default {
       // insert the button inside the plyr instance so that it shows up in fullscreen mode
       this.$nextTick(() => {
         let maximizeButton = document.getElementById("plioMaximizeButton");
-        if (maximizeButton != undefined) this.mountOnFullscreenPlyr(maximizeButton);
+        if (maximizeButton != undefined)
+          this.mountOnFullscreenPlyr(maximizeButton);
       });
     },
+    isMovingToTimestampAllowed(timestamp) {
+      if (this.firstUnansweredItem == null) return true;
+      // check whether skip is disabled and all items have been answered
+      return timestamp < this.firstUnansweredItem.time;
```

Comment:
here the comment says that `check whether skip is disabled...` but that check is not happening in the code?